src/posit/connect/groups.py

```diff
@@ -10,18 +10,83 @@
 if TYPE_CHECKING:
     import requests
 
+    from posit.connect.context import Context
+
+    from .users import User
```

Comment:
Do you need this and line 58 below? Is that just an artifact of how type checking works?